### ISSUE TITLE
StreamingTDSModelConverter.cpp:91] Invalid params for linear.

### ISSUE THREAD

**Issue by AVSuni** [facebookresearch/wav2letter#627]
Hi, 

I'm trying to convert a streaming convnets model that is identical in architecture to the pretrained English model. I get the following error 

`I0429 11:37:03.335691 20124 StreamingTDSModelConverter.cpp:179] Number of classes (network): 9998
Skipping View module: V -1 NFEAT 1 0
Skipping SpecAugment module: SAUG 80 27 2 100 1.0 2
Skipping Dropout module: DO 0.1
Skipping Dropout module: DO 0.1
Skipping Dropout module: DO 0.1
Skipping Dropout module: DO 0.1
Skipping Reorder module: RO 2 1 0 3
Skipping View module: V 2160 -1 1 0
F0429 11:37:06.093279 20124 StreamingTDSModelConverter.cpp:91] Invalid params for linear.
*** Check failure stack trace: ***
    @     0x7f166e74f0cd  google::LogMessage::Fail()
    @     0x7f166e750f33  google::LogMessage::SendToLog()
    @     0x7f166e74ec28  google::LogMessage::Flush()
    @     0x7f166e751999  google::LogMessageFatal::~LogMessageFatal()
    @     0x55e82a59db13  (anonymous namespace)::convertLinear()
    @     0x55e82a531e87  main
    @     0x7f16665a9b97  __libc_start_main
    @     0x55e82a59a97a  _start
Aborted (core dumped)`

Any thoughts? Many thanks.

**Comment by vineelpratap** [comment]
Hi, could you paste the your arch file here

**Comment by AVSuni** [comment]
Thanks @vineelpratap . I used the `am_500ms_future_context.arch` from the streaming convnets recipe without any modifications. After a long fight, conversion ended up working on another machine. I will try to go back and see the differences between my Dockerfiles.

**Comment by tranmanhdat** [comment]
i met same probem like that, how can i fix it?
![Screenshot from 2021-01-18 15-22-52](https://user-images.githubusercontent.com/30494878/104889632-0f865280-59a1-11eb-9106-9149ba149158.png)
my arch file in the link [turorial](https://github.com/facebookresearch/wav2letter/blob/v0.2/recipes/models/streaming_convnets/librispeech/am_500ms_future_context.arch)

**Comment by tlikhomanenko** [comment]
cc @vineelpratap

**Comment by tranmanhdat** [comment]
i findout that error come when i train in a computer and convert in another computer
